### PR TITLE
[DR-7840-Implement-breadcrumb-component] 

### PR DESCRIPTION
**Pull request by ghost**
adds custom breadcrump style

### CODE REVIEW

File: scss/_custom.scss [hunk 1]
```diff
@@ -970,10 +970,11 @@ $breadcrumb-padding-y:          .75rem !default;
 $breadcrumb-padding-x:          1rem !default;
 $breadcrumb-item-padding:       .5rem !default;
 
-$breadcrumb-bg:                 $gray-lighter !default;
+$breadcrumb-bg:                 transparent !default;
```
Comment (jayhof) [inline scss/_custom.scss]: Only variables using the default configuration value should use the "!default" mark

File: scss/_custom.scss [hunk 2]
```diff
@@ -970,10 +970,11 @@ $breadcrumb-padding-y:          .75rem !default;
...
+$breadcrumb-item-color:         $gray-light !default;
 $breadcrumb-divider-color:      $gray-light !default;
 $breadcrumb-active-color:       $gray-light !default;
-$breadcrumb-divider:            "/" !default;
+$breadcrumb-divider:            " " !default;
```
Comment (jayhof) [inline scss/_custom.scss]: "_blank" in the ticket description was a mark to ask @maxstrebel for the character he'd like to use as a breadcrumb separator

File: scss/custom/_breadcrumb.scss [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  .breadcrumb-item {
+    color: $breadcrumb-item-color;
+    text-decoration: underline;
+
+    &.active {
+      text-decoration: none;
+    }
+  }
+
+  li.breadcrumb-item a {
```
Comment (jayhof) [inline scss/custom/_breadcrumb.scss]: The breadcrumb items cannot only be nested inside a list element, it's also possible to use a-tags within a nav your style declaration should follow this idea.

Comment (maxstrebel) [reply scss/custom/_breadcrumb.scss]: In general: if you can style a thing only by using a class, try to do it. So you can overwrite it easier when you need to do. Less specific is always best.

So in this case try `.breadcrumb-item { ...` first and if it is enough, be happy :)

**Review by jayhof — APPROVED** [review]
Thanks!

**Review by maxstrebel — APPROVED** [review]
Thx Roland

**Pull request merged**